### ISSUE TITLE
Unable to replicate the experiment results with this repo

### ISSUE THREAD

**Issue by gmyofustc** [Hxyz-123/Font-diff#15]
Generated images are completely wrong content, hope you guys do not waste time on this repo

**Comment by mscb402** [comment]
Yes, I was give up. I spent a lot of time training, but it didn't work for me. It seems like I'm not the only one unable to replicate the experimental results.
